stock/templates/registered/hrapplicationview.html

```diff
@@ -1,223 +1,111 @@
+                                    <h4 class="panel-title">
+                                        <a class="collapsed" data-toggle="collapse" data-parent="#accordion"
+                                            href="#collapseThree" aria-expanded="false"
+                                            aria-controls="collapseThree">
+                                            Comments - {{ comments|length }}
                                         </a>
-                                        <button type="button" class="btn btn-primary btn-lg" data-toggle="modal"
-                                                data-target="#myModal">
-                                            Comment
-                                        </button>
+                                    </h4>
+                                </div>
+                                <div id="collapseThree" class="panel-collapse collapse" role="tabpanel"
+                                    aria-labelledby="headingThree">
+                                    <div class="panel-body">
+                                        {% for comment in comments %}
+                                            <div class="panel panel-default">
+                                                <div class="panel-heading" role="tab" id="">
+                                                    <div class="panel-title">{{comment.created}} - {{ comment.user }}</div>
+                                                </div>
+                                                <div class="panel-body">{{ comment.text }}</div>
+                                            </div>
+                                        {% endfor %}
                                     </div>
                                 </div>
-
-                                <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
-                                    <div class="panel panel-default">
-                                        <div class="panel-heading" role="tab" id="headingThree">
-                                            <h4 class="panel-title">
-                                                <a class="collapsed" data-toggle="collapse" data-parent="#accordion"
-                                                   href="#collapseThree" aria-expanded="false"
-                                                   aria-controls="collapseThree">
-                                                    Comments - {{ comments|length }}
-                                                </a>
-                                            </h4>
+                            </div>
+                        </div>
+                        {% if perms.hrapplications.add_applicationcomment %}
+                            <div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"
+                                aria-hidden="true">
```

Comment:
Oh god that's what a modal is. I needed to include the add button to make it pop up. For some reason I had it in my head that the old system had an in-line comment form.

It's been corrected.